lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -9,6 +10,8 @@ class FirebaseAppDistributionApiClient
       MAX_POLLING_RETRIES = 60
       POLLING_INTERVAL_SECONDS = 2
 
+      include Auth::FirebaseAppDistributionAuthClient
```

Comment:
I believe both of those options are valid. Putting the auth client inside the main action allows us to pass in an auth token in the constructor. The only difference is that now the main action extends the auth client instead of being included in the API client. This is because it is still a module. Passing in an auth token in the constructor is a good idea so that we can also get rid of the auth_token method